src/main.ts

```diff
@@ -250,6 +255,7 @@ async function getTsServerRepoResult(
 
         console.log(`Issue found in ${newTsServerPath} (new):`);
         console.log(insetLines(prettyPrintServerHarnessOutput(newSpawnResult.stdout, /*filter*/ false)));
+        await fs.promises.writeFile(rawErrorsScriptPath, prettyPrintServerHarnessOutput(newSpawnResult.stdout, false));
```

Comment:
Can you please add a comment with the parameter name, like the line above?  We like to do that for literal arguments.